vignettes/faq.Rmd

```diff
@@ -82,4 +82,31 @@ We particularly highlight two functions which might be useful:
 For an example use of these functions, have a look at the [`epidist-paper`](https://github.com/parksw3/epidist-paper) repository containing the code for @park2024estimating.
 (Note that in that codebase, we use `map` as a part of a [`targets`](https://books.ropensci.org/targets/) pipeline.)
 
+## How did you choose the default priors for `epidist`?
+
+`brms` provides default priors for all parameters.
+However, some of those priors do not make sense in the context of our application.
+Instead, we used [prior predictive checking](https://mc-stan.org/docs/stan-users-guide/posterior-predictive-checks.html) to set `epidist`-specific default priors which produce epidemiological delay distribution mean and standard deviation parameters in a reasonable range.
+
+For example, here are summary statistics for the distributions on the delay distribution mean and standard deviation, as implied by the following prior distributions on the `brms` lognormal `mu` and `sigma` parameters
+$$
+\mu \sim \mathcal{N}(2, 0.5^2), \\
```

Comment:
I'm not convinced we want to have an additional point of truth for our priors we want to keep updated but happy to keep for now @athowes